examples/io/src/read_file.cc

```diff
@@ -48,7 +48,8 @@ struct read_fn {
     // Compute the absolute offsets to the section that this reader task
     // is supposed to read from the file
     int64_t my_lo = my_id * size / num_readers;
-    int64_t my_hi = std::min((my_id + 1) * size / num_readers, size);
+    int64_t my_hi =
+      std::min(static_cast<int64_t>((my_id + 1) * size / num_readers), static_cast<int64_t>(size));
```

Comment:
I think it is safe to cast just `size` to `int64_t` here. i.e.
```c++
auto ssize = static_cast<int64_t>(size);
int64_t my_hi =
      std::min((my_id + 1) * ssize / num_readers, ssize);
```

Reply:
@CharlelieLrt could you please check if this change works for you also? I can't check myself :-(

Reply:
Yes, it works.

Reply:
fe0205d